src/nxdoc/web-ui/web-ui-release-notes.md

```diff
@@ -22,16 +22,17 @@ Starting from Web UI version `3.1.6` released in March 2024, functional tests wr
 
 ## Recently Released Changes
 
-{{{multiexcerpt 'web-ui-updates' page='web-ui-release-notes-3-1-8'}}}
+{{{multiexcerpt 'web-ui-updates' page='web-ui-release-notes-3-1-9'}}}
 
 ---
 
 ## Previous Release Notes
 
-<!-- | [Web UI 3.1.8]({{page page='web-ui-release-notes-3-1-8'}})                  | Accessibility improvements and bugfix release     | -->
+<!-- | [Web UI 3.1.9]({{page page='web-ui-release-notes-3-1-9'}})                  | Veracode issue fix and bugfix release     | -->
```

Comment:
```suggestion
<!-- | [Web UI 3.1.9]({{page page='web-ui-release-notes-3-1-9'}})                  |  Security and bugfix release     | -->
```

Customers may not know that veracode is a code scanning tool, we need to mention it as security for everyone to understand.